src/main/java/com/netgrif/application/engine/history/domain/caseevents/DeleteCaseEventLog.java

```diff
@@ -5,11 +5,15 @@
 import lombok.EqualsAndHashCode;
 import org.springframework.data.mongodb.core.mapping.Document;
 
+@Document(collection = "eventLogs")
```

Comment:
This may be a breaking change and may requires migration. Is this change necessary?

Reply:
This should not be breaking change because there is only one collection `eventLogs` right now.